grpc_ros_adapter/services/frame_service.py

```diff
@@ -5,6 +5,7 @@
 from protobuf import tf_pb2
 
 import utils.ros_handle as rh
+import utils.extensions
```

Comment:
Is this necessary?

Reply:
yes, as_ros() extension methods require it